src/main/java/teammates/ui/webapi/ActionFactory.java

```diff
@@ -50,6 +50,7 @@ public final class ActionFactory {
         map(ResourceURIs.ACCOUNT_REQUEST, GET, GetAccountRequestAction.class);
         map(ResourceURIs.ACCOUNT_REQUEST, POST, CreateAccountRequestAction.class);
         map(ResourceURIs.ACCOUNT_REQUEST, DELETE, DeleteAccountRequestAction.class);
+        map(ResourceURIs.ACCOUNT_REQUESTS_PENDING, GET, GetPendingAccountRequestsAction.class);
```

Comment:
I'm fine with this change. 
I think if ever in future we need an endpoint to get all account requests, likely will have to add back the `/pending` (to follow the precedent for existing classes, like `ResourceURIs.SESSIONS_ONGOING` and `ResourceURIs.NOTIFICATION_READ`)

Reply:
How about remove the `/pending` at the end AND add a query param? e.g. `status` where the value can only be `pending=0` for now? If the status param receives other enum value then throw an error. In this way it would be more extensible in the future.
Btw Notification Read is a bit different from this case as ReadNotification is an entity alone.

Reply:
>`ResourceURIs.SESSIONS_ONGOING`

I've seen this before, and while it is there, I am not sure if it's right. 😅 To me, it probably needs to be refactored and just merged into `GetFeedbackSessionsAction`, but I think this is probably out of our scope. I have a feeling this was done like this because `GetFeedbackSessionsAction` is a normal `Action` while `GetOngoingSessionsAction` is an `AdminOnlyAction`, and it was difficult to do the access control, but I don't really know.

>AND add a query param

Whether you necessitate it or not, as long as it's a query param, I think I would be supportive of it.

>e.g. `status`

Yes, if you want to add the query param, I think it can be named `status`.

>where the value can only be `pending=0`

Do you mean like `/account/requests?status=0`? Maybe the string `pending` itself would be fine, and maybe clearer, like `/account/requests?status=pending`?

Reply:
Yep `/account/requests?status=pending`  look good.
@EuniceSim142 let’s refactor to this format?